html/elements/link.json

```diff
@@ -780,6 +780,42 @@
               }
             }
           },
+          "compression-dictionary": {
+            "__compat": {
+              "description": "rel=compression-dictionary",
```

Comment:
```suggestion
              "description": "rel=\"compression-dictionary\"",
```

Reply:
Yeah I would have gone with that too but the next line (`rel=expect`) doesn't have quotes so tried to stick with that convention. Should I change that too? Or do you only need quotes due to the hyphen?

Reply:
Oh, you're right, we seem to use quotes only when necessary. I peeked at the previous item, which contains spaces. All good. 🙂